modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -624,4 +633,44 @@ private void addEncoderAndDecoder(Map<String, Object> vendorExtensions, String d
         ELM_018,
         ELM_019
     }
+
+    @Override
+    public void postProcessFile(File file, String fileType) {
+        if (file == null) {
+            return;
+        }
+
+        String elmFmtPath = System.getenv("ELM_FORMAT_PATH");
+        if (StringUtils.isEmpty(elmFmtPath)) {
+            return; // skip if ELM_FORMAT_PATH env variable is not defined
+        }
+
+        // only procees the following type (or we can simply rely on the file extension to check if it's a elm file)
+        Set<String> supportedFileType = new HashSet<String>(
+                Arrays.asList(
+                        "supporting-mustache",
+                        "model-test",
+                        "model",
+                        "api-test",
+                        "api"));
+        if (!supportedFileType.contains(fileType)) {
+            return;
+        }
+
+         // only process files with elm extension
+        if ("elm".equals(FilenameUtils.getExtension(file.toString()))) {
+            // currently only support "elm-format -w yourcode.elm"
+            String command = elmFmtPath + " --yes " + file.toString();
```

Comment:
I don't think we need `--yes`.
I think we do need `--elm-version=0.18` for Elm 0.18 (see https://github.com/avh4/elm-format)

Reply:
I've to keep `--yes`. Otherwise, the command execution won't complete

Reply:
OK, and it seems `elm-format` did not succeed in your last commit. Are you running the latest version?

Reply:
Updated via d3183af